templates/project-ts/src/interact.ts

```diff
@@ -82,6 +82,19 @@ Success! Update transaction sent.
 
 Your smart contract state will be updated
 as soon as the transaction is included in a block:
-https://berkeley.minaexplorer.com/transaction/${sentTx.hash()}
+${getTxnUrl(config.url, sentTx.hash())}
 `);
 }
+
+function getTxnUrl(graphQlUrl: string, txnHash: string | undefined) {
+  const explorerName = new URL(graphQlUrl).hostname
```

Comment:
```suggestion
```

Reply:
Similar comment made in the `deploy.js` file about removing this.

Reply:
Resolving then.